src/main/kotlin/racingCar/domain/Race.kt

```diff
@@ -1,15 +1,20 @@
-package racingcar.domain
+package racingCar.domain
 
 class Race(
     private val tryCount: TryCount,
     private val cars: Cars,
     private val raceResult: RaceResult
 ) {
 
+    private var _isFinished = false
+    val isFinished: Boolean
+        get() = _isFinished
```

Comment:
아래 처럼 구현하는것도 가능할것 같아요 😄 
```suggestion
var finished: Boolean = false 
  private set 
```

Reply:
혹시 `isFinished` 대신 `finished`라는 네이밍을 사용하신 이유가 따로 있으신가요~?!

Reply:
`is` 라는 prefix는 getter를 정의할떄 사용하는것으로 알고 있습니다 😄 
프로퍼티를 정의하는 경우는 생략하고 있어요 !